oracle/oracle-explanation/canonical-offerings.rst

```diff
@@ -0,0 +1,62 @@
+
+* **Minimal images:** These are minimal images designed for automated deployment at scale and have a reduced default package set. They are much smaller, boot faster, and require fewer security updates over time due to the fewer installed packages.
+
+Each of these images have variants for AMD64 and ARM64 (Ampere processors). The minimal images for ARM64 are only available from Ubuntu 22.04 onwards. 
+
+The images are built regularly and follow a thorough test framework that helps detect any issues / regressions due to package changes. They are then shared with Oracle privately who publish them on the cloud platform once per month.
+
+With a single image type that is validated to work on both virtual machines and bare metal instances, the images aim to provide a consistent experience for users.
+
+Although only Ubuntu LTS (Long-Term Support) images are publicly available on Oracle Cloud, Canonical also builds Oracle-optimised images for non-LTS releases. This allows for continuous internal testing and ensures that future LTS releases work well on Oracle Cloud.
+
+For instructions on how to find the images, refer to :doc:`../oracle-how-to/find-ubuntu-images`.
+
+
+Optimisations for Oracle Cloud
+------------------------------
+
+Generally the images are built in a qcow2 format and support the following options:
+
+* Boot firmware - boot with BIOS or UEFI (preferred). ARM64 images only support UEFI boot.
+* Launch modes - support for both PARAVIRTUALIZED and NATIVE launch modes.  
+* Network interfaces - support for VFIO and PARAVIRTUALIZED modes.
+* Boot volume -  allows booting through iSCSI or PARAVIRTUALIZED storage.
+
+The customisations include NVMe tunings, iSCSI configurations to allow instances to boot from iSCSI backend when appropriate, logging improvements, iptables and udev configurations specific to Oracle Cloud platform, among others.
+
+
+Kernel optimisations
+--------------------
+
+Ubuntu images available on Oracle Cloud run the linux-oracle kernel. Additional kernel variants are also offered by Canonical, that users can install as appropriate.
+
+By default Ubuntu images use a **rolling kernel model**, which provides the latest upstream bug fixes and performance improvements around task scheduling, I/O scheduling, networking, hypervisor guests and containers. A rolling kernel model transitions the default linux-oracle kernel from one base version to the next as part of its regular patching cycle. That new kernel, called the HWE Kernel (Hardware Enablement Kernel) is the kernel of the latest Ubuntu release.
+
+As an example, users running Ubuntu 22.04 instances that were launched back in 2022, would run the 5.15 kernel by default. But as new Ubuntu versions got released, the linux-oracle kernel rolled on to the next release's kernel version. So an Ubuntu 22.04 instance launched at a later point in time, would include different kernel versions depending on when it was launched - it could be version 6.5 or and more recently 6.8.
```

Comment:
That plus maybe we want to explicitly state that running instances will have their kernel updated automatically (upon reboot).

Reply:
+1

Reply:
Added that.